sstmac/software/launch/launch_event.h

```diff
@@ -125,12 +125,15 @@ class StartAppRequest :
      TaskId tid,
      NodeId to,
      NodeId from,
+#if SSTMAC_INTEGRATED_SST_CORE
```

Comment:
This is kind of painful, maybe we should talk about a way to do this differently.

Reply:
Again, keep fragile code contained to sst-core.

Reply:
I meant that changing the type of the params depending on which PDES we use is a bit weird.   If this is just for storing info for the object to use it'd be nicer to swap to json and update the serialization.  But 🤷‍♀️ it's not one of the changes I'm requesting.  I just think this method is a bit error prone.